src/libs/ReportUtils.ts

```diff
@@ -6113,7 +6113,8 @@ function isValidReportIDFromPath(reportIDFromPath: string): boolean {
 /**
  * Return the errors we have when creating a chat or a workspace room
  */
-function getAddWorkspaceRoomOrChatReportErrors(report: OnyxEntry<Report>): Errors | null | undefined {
+function getAddWorkspaceRoomOrChatReportErrors(reportOrID: OnyxEntry<Report> | string): Errors | null | undefined {
+    const report = typeof reportOrID === 'string' ? ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportOrID}`] : reportOrID;
```

Comment:
We shouldn't fetch the report in the lib here if we don't need to. Can we just get the report in the `MoneyRequestView` with `useOnyx` and pass it in?

Reply:
Done